From da7aa5500205845fcdb75f4d18591a9f7ce1ad19 Mon Sep 17 00:00:00 2001
From: Igal Zeltser <izeltser@habana.ai>
Date: Thu, 17 Feb 2022 17:35:23 +0200
Subject: [PATCH] Revert "[SW-0] gaudi2: create serdes nic fw file"

This reverts commit 6fa42702c2e06e09e0c42021299aa4f2ee623f61.

this commit breaks the nic init sequence in u-boot,
reverted for rework
---

diff --git a/board/habanalabs/gaudi2/post-image-agents.sh b/board/habanalabs/gaudi2/post-image-agents.sh
index bd2253c..2567bf1 100755
--- a/board/habanalabs/gaudi2/post-image-agents.sh
+++ b/board/habanalabs/gaudi2/post-image-agents.sh
@@ -14,13 +14,7 @@
 HL_PREBOOT_VERSION_SECUREBOOT=`cat ${BR2_EXTERNAL}/output/${HL_DEV}_arc/build/zephyr-custom/build-preboot/buildversion-habanalabs`
 ITS_SED_PARAMS+="s/@HL_PREBOOT_VERSION_SECUREBOOT@/${HL_PREBOOT_VERSION_SECUREBOOT}/g;"
 
-NIC_FW_FILE=${BR2_EXTERNAL}/board/habanalabs/${HL_DEV}/fw/condor.fw.bin
-AGENT_SERDES_FW_NAME=zephyr_agentfw_serdes.bin
-
 function handle_fw_copy {
-
-	# Set NIC FW to be the serdes agent FW file
-	cp ${NIC_FW_FILE} ./${AGENT_SERDES_FW_NAME}
 
 	# Copy common arc files required for agent fit creation
 	handle_fw_copy_arc
diff --git a/board/habanalabs/gaudi2/post-image-fit.sh b/board/habanalabs/gaudi2/post-image-fit.sh
index ed8984d..2935614 100644
--- a/board/habanalabs/gaudi2/post-image-fit.sh
+++ b/board/habanalabs/gaudi2/post-image-fit.sh
@@ -5,6 +5,8 @@
 KERNEL_START_ADDR_HI=0x00000080
 KERNEL_LOAD_ADDR_LO=0x08080000
 
+NIC_FW_FILE=${BR2_EXTERNAL}/board/habanalabs/${HL_DEV}/fw/condor.fw.bin
+
 # Uboot load address in ITS file
 if [ ${HL_TARGET} == "onix" ]; then
 UBOOT_ADDR_LO=0x00100000
